src/@types/mediawiki.ts

```diff
@@ -0,0 +1,9 @@
+export interface MwMessage {
```

Comment:
Maybe it is still a bit too early, but I wonder if at some point we should upstream all these types to https://github.com/wikimedia/typescript-types/blob/master/MediaWiki.d.ts

Reply:
I created https://github.com/wikimedia/typescript-types/pull/19 during a meeting earlier today, let’s see if anything happens there.